dockerfiles/Dockerfile.sidekiq

```diff
@@ -0,0 +1,21 @@
+FROM ruby:2.4-alpine
+LABEL maintainer="Josh Ashby <me@joshisa.ninja>"
+
+RUN apk add --no-cache --update build-base postgresql-dev git curl
+
+RUN echo -e 'http://dl-cdn.alpinelinux.org/alpine/edge/main\nhttp://dl-cdn.alpinelinux.org/alpine/edge/community\nhttp://dl-cdn.alpinelinux.org/alpine/edge/testing' > /etc/apk/repositories && \
+    apk add --no-cache nodejs-current yarn
+
+RUN mkdir /app
+RUN mkdir /dropzone
+WORKDIR /app
+
+COPY Gemfile Gemfile.lock ./
+RUN echo "install: --no-document" > $HOME/.gemrc && echo "update: --no-document" >> $HOME/.gemrc
```

Comment:
you should have your own base image which does this

Reply:
I did consider building a base image. Steps for that would be to make another dockerfile for it and then publish it to a registry, which these would then use, correct?

Reply:
Yeah. I should say that at times I've been bitten by that. You can get into a spot where docker (the daemon) doesn't pull the new base image from docker (the hub) and then your docker (containers) refuse to be more up to date. Docker (the command) has some purge sub-commands that are helpful here.